flask_ldap_login/__init__.py

```diff
@@ -148,19 +154,24 @@ def bind_search(self, username, password):
             log.debug("Binding with the BIND_DN %s" % user)
             self.conn.simple_bind_s(user, bind_auth)
         except ldap.INVALID_CREDENTIALS:
-            log.debug("Could not connect bind with the BIND_DN=%s" % user)
+            msg = "Could not connect bind with the BIND_DN=%s" % user
+            log.debug(msg)
+            if self._raise_errors:
+                raise ldap.INVALID_CREDENTIALS(msg)
```

Comment:
That's part of the reason for this PR - help in the configuration process on installs.  So should we create our own exception that more accurately reflects the problem?